onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -166,8 +166,8 @@ contract CartesiDAppTest is TestBase {
         // not able to execute voucher because dapp has 0 balance
         assertEq(erc20Token.balanceOf(address(dapp)), 0);
         assertEq(erc20Token.balanceOf(recipient), 0);
-        bool success = executeVoucher(voucher, proof);
-        assertEq(success, false);
+        vm.expectRevert("ERC20: transfer amount exceeds balance");
+        executeVoucher(voucher, proof);
         assertEq(erc20Token.balanceOf(address(dapp)), 0);
         assertEq(erc20Token.balanceOf(recipient), 0);
```

Comment:
For example, we don't need to check that the balances haven't changed because the call to `executeVoucher` was not successful, which means there were no state changes.
This occurs in many places throughout this file.